src/main/java/network/brightspots/rcv/TallyTransfers.java

```diff
@@ -24,20 +24,40 @@
 
 package network.brightspots.rcv;
 
+import static network.brightspots.rcv.Utils.isNullOrBlank;
+
 import java.math.BigDecimal;
 import java.util.HashMap;
 import java.util.Map;
 
 // TallyTransfers class stores summary info on vote transfers
-// used primary as visualizer input to help build Sankey plots
+// used primarily as visualizer input to help build Sankey plots
 class TallyTransfers {
 
+  static final String EXHAUSTED = "exhausted";
+  static final String RESIDUAL_TARGET = "residual surplus";
+  static final String UNCOUNTED = "uncounted";
```

Comment:
Should we include a test with "uncounted"?

Reply:
"uncounted" never actually appears in the output